README.md

```diff
@@ -70,6 +70,14 @@ dependencies {
 }
 ```
 
+Please also add the following lines to your repositories section in your gradle script:
```

Comment:
Does this also need to go in our samples and docs? cc: @hamiltonha

Reply:
I believe it will need to go in the sample for sure...I don't think the existing sample will work without the change.